app/assets/javascripts/services/post_service.js

```diff
@@ -54,5 +35,18 @@ ManageIQ.angular.app.service('postService', ['miqService', '$timeout', '$window'
       $window.location.href = redirectURL;
     });
   };
+
+  function postSuccess(response, successMsg, redirectURL) {
+    $timeout(function () {
+      if (response.error) {
+        var msg = __(response.error.klass + ': ' + response.error.message);
```

Comment:
This is not something we can gettext, maybe it makes sense to do `response.error.klass + ": " + __(response.error.message)` (assuming the messages come untranslated but are in the translation catalogue), but not the whole thing.